testsuites/CBLTester/topology_specific_tests/multiple_sync_gateways/test_sgreplicate_cbl.py

```diff
@@ -0,0 +1,2139 @@
+    sgw_cluster2_bucket_3 = 'listener_tests/sg_replicate_sgw_cluster_databucket_3'
+    sg_db3 = "sg_db3"
+    name3 = "autotest3"
+    sg_client = MobileRestClient()
+    sgwgateway = SyncGateway()
+
+    # 1. Have 3 sgw nodes and have 3 cbl db
+    db, num_of_docs, sg_db1, sg_db2, name1, name2, password, channels1, _, replicator, _, replicator_authenticator2, _, sg2_blip_url, sg1, sg2, repl1, c_cluster, cbl_db1, cbl_db2 = setup_syncGateways_with_cbl(params_from_base_test_setup, setup_customized_teardown_test, cbl_replication_type="push_pull", sg_conf_name=sg_conf_name, sgw_cluster1_sg_config_name=sgw_cluster1_conf_name,
+                                                                                                                                                                                                                 sgw_cluster2_sg_config_name=sgw_cluster2_conf_name)
+    sg3 = c_cluster.sync_gateways[2]
+    sgw_cluster1_sg_config = sync_gateway_config_path_for_mode(sgw_cluster2_bucket_3, sg_mode)
+    sgw_cluster1_config_path = "{}/{}".format(os.getcwd(), sgw_cluster1_sg_config)
+    sgwgateway.redeploy_sync_gateway_config(cluster_config=cluster_config, sg_conf=sgw_cluster1_config_path, url=sg3.ip,
+                                            sync_gateway_version=sync_gateway_version, enable_import=True)
+    authenticator = Authenticator(base_url)
+
+    sg3_blip_url = "ws://{}:4984/{}".format(sg3.ip, sg_db3)
+    if sg_ssl:
+        sg3_blip_url = "wss://{}:4984/{}".format(sg3.ip, sg_db3)
+    sg_client.create_user(sg3.admin.admin_url, sg_db3, name3, password=password, channels=channels1)
+    cookie, session_id = sg_client.create_session(sg3.admin.admin_url, sg_db3, name3)
+    replicator_authenticator3 = authenticator.authentication(session_id, cookie, authentication_type="session")
+
+    # 2. Create docs in cbd db1 and cbl db3.
+    db.create_bulk_docs(num_of_docs, "Replication2", db=cbl_db1, channels=channels1)
+    db.create_bulk_docs(num_of_docs, "Replication3", db=cbl_db3, channels=channels1)
+    # 3. Start push_pull, continuous replicaation cbl_db2 <-> sg2, cbl_db3 <-> sg3
+    repl2 = replicator.configure_and_replicate(
+        source_db=cbl_db2, replicator_authenticator=replicator_authenticator2, target_url=sg2_blip_url,
+        replication_type="push_pull", continuous=True)
+
+    repl3 = replicator.configure_and_replicate(
+        source_db=cbl_db3, replicator_authenticator=replicator_authenticator3, target_url=sg3_blip_url,
+        replication_type="push_pull", continuous=True)
+
+    # 4. start replication on sg1 push_pull from sg1<->sg3
+    sgw_replid_1 = sg1.start_replication2(
+        local_db=sg_db1,
+        remote_url=sg3.url,
+        remote_db=sg_db3,
+        remote_user=name3,
+        remote_password=password,
+        direction="pushAndPull",
+        continuous=continuous
+    )
+
+    # 4. start replication on sg2 push_pull from sg2<->sg3
```

Comment:
"# 5." ? summary says 5. start replication on sg3, in code is sg1 and sg2. update the summary?